docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -1,319 +1,402 @@
+::
+  
+  file:bytes:mime:pe:pdbpath
 
 
-.. storm-pre:: [inet:dns:a = (woot.com, 1.2.3.4)]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:dns:a = (woot.com, 1.2.3.4)
+.. _lift-prop-univ:
 
+Lift by Universal Property
+++++++++++++++++++++++++++
 
-*Lift a specific GUID node:*
+**Syntax:**
 
-- Lift the organization node with the specified GUID:
+*<form>* **.** *<prop>*
 
+**.** *<prop>*
 
-.. storm-pre:: [ou:org=2f92bc913918f6598bcf310972ebf32e]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: ou:org=2f92bc913918f6598bcf310972ebf32e
+A :ref:`gloss-universal-prop` applies to and can be used by any node.
 
+**Examples:**
 
-*Lift a specific digraph (edge) node:*
+*Lift all email (inet:email) nodes with a .seen property:*
 
-- Lift the ``edge:has`` node linking the person node representing "Bob Smith" to his email address:
+.. storm-pre:: [ (inet:email=dns@jomax.net .seen=(2016/12/10 08:56:23, 2023/10/20 14:57:53)) (inet:email=bolekrejci@centrum.cz .seen=(2017/10/12 00:00:00, 2017/10/12 00:00:00.001))]
+.. storm-pre:: inet:email.seen
+::
+  
+  inet:email.seen
 
+*Lift all nodes with a .seen property:*
 
-.. storm-pre:: [edge:has=((ps:person,12af06294ddf1a0ac8d6da34e1dabee4),(inet:email, bob.smith@gmail.com))]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: edge:has=((ps:person,12af06294ddf1a0ac8d6da34e1dabee4),(inet:email, bob.smith@gmail.com))
+.. storm-pre:: .seen
+::
+  
+  .seen
 
+*Lift all nodes in Synapse:*
 
-*Lift by the presence of a secondary property (<prop>):*
+.. storm-pre:: .created
+::
+  
+  .created
 
-- Lift the DNS SOA record nodes that have an email property:
+**Usage notes:**
 
+- The ``.seen`` universal property can optionally be used by any node.
+- To lift any node (i.e., nodes of any form) that **have** a universal property set, you only need to specify
+  the property name with its leading dot (e.g., ``.seen``).
+- Because the ``.created`` property is automatically set for every node when it is first added to Synapse,
+  lifting by the ``.created`` property effectively lifts every node in Synapse (technically, every node in the
+  current :ref:`gloss-view`).
 
-.. storm-pre:: [inet:dns:soa="*" :email=happygogo@yahoo.net]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:dns:soa:email
 
+.. _lift-prop-extend:
 
-*Lift by a specific property value (<prop> = <pval>):*
+Lift by Extended Property
++++++++++++++++++++++++++
 
-- Lift the organization node with the alias ``vertex``:
+**Syntax:**
 
+*<form>* **:_** *<prop>*
 
-.. storm-pre:: [ou:org="*" :alias=vertex]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: ou:org:alias = vertex
+An :ref:`gloss-extended-prop` is a custom property that has been added to the Synapse data model to capture specialized data
+for a given form. To avoid potential namespace collisions with standard properties, we recommend naming extended properties
```

Comment:
Extended props are required to begin with an underscore, not just a recommendation